src/scss/sections/_header.scss

```
@@ -0,0 +1,38 @@
+.img-logo-adaptiv {
+  max-width: 100%;
+  height: auto;
+  @include adaptiv-img(155, 58);
+}
+
+.phone-number {
+  padding-left: 12px;
+}
+
+.color-accent {
+  color: $color-accent;
+}
+
+.header-page {
+  background: url("../assets/img/first-page-bg.jpg");
+  background-position: right 65%;
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 100%;
+}
+
+.first-page {
+  &__action {
+    max-width: 300px;
+    background: $color-white-bg;
+  }
+}
+
+@include media-breakpoint-down(sm) {
+  .header-page {
+    background: url(../assets/img/first-page-bg-xs.png);
+    background-position: bottom;
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 100%;
+  }
```

Changes only `src` and `position`, get rid of other properties. 
```suggestion
  .header-page {
    background: url(../assets/img/first-page-bg-xs.png);
    background-position: bottom;
  }
```

---

background-size: cover; - need to save because you will get many background images instead of just one

---

It's a duplicate prop, remove `background-size: cover;`